CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ## [Unreleased]
 
+## [3.4.1] - 2020-05-23
```

Comment:
```suggestion
```

Leaving this comments under `## [Unreleased]` is ok. We will add a version number (and a release date) when an actual release will be done.

Reply:
Done!